services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/CryptoProvider.java

```diff
@@ -114,8 +113,7 @@ public Certificate getCertificate() {
       try (InputStream certStream = certResource.getInputStream()) {
         this.certificate = getCertificateFromStream(certStream);
       } catch (IOException | CertificateException e) {
-        logger.error("Failed to load certificate from {}", certificatePath, e);
-        throw new RuntimeException(e);
+        throw new RuntimeException("Failed to load certificate from " + certificatePath, e);
```

Comment:
For handling the `IOException`, we propose the less generic `UncheckedIOException` which extends `RuntimeException` and also wraps an `IOException`

Reply:
UncheckedIOException is not compatible here. Is it likely that you want to distinguish between RuntimeException and UncheckedIOException? If not I would say it only makes the code unnecessarily more complex. For the cases where UncheckedIOException is compatible I also don't any benefit of the more specific exception.

Reply:
Replaced with UncheckedIOException and RuntimeException in case of CertificateException.

Reply:
Just checked with Steve and we agree that sticking with the initial ``} catch (IOException | CertificateException e) {`` works fine for us here.

If you don't mind reverting this particular change, we can go ahead and merge.

Reply:
Sure